testdata/package/example/1.0.0/manifest.yml

```diff
@@ -10,6 +10,9 @@ release: ga
 
 owner.github: "ruflin"
 
+conditions:
```

Comment:
@ph @michalpristas I initially was thinking about aligning the syntax with the constraints on the agent. But it only complicates things. The reason is that the available conditions here are very limited and have very specific checks. Adding additional "non yaml" syntax only complicates things. So here is what I'm proposing at the moment.

As each `condition` can only be used once I made it a key/value pair instead of an array.

One thing I think we should align is the naming. I landed on conditions as I think it fits best in all the places. WDYT about renaming it also in the agent?

Reply:
+1 on using "conditions" instead of constraints, which would align with dynamic input.

Now, I am Okayish with using key values pair, for what you are proposing.

1. Its a limited set of fixed conditions.
2. The conditions are joined with an **AND** this mean all condition predicates must return true.

Reply:
Also, using key/value also mean you don't have to have the parser in Kibana and we don't have to share code in the registry. 

So since everything is defined upfront I do not see a concern if the syntax arent the same, it's easier to start fixed and limited than to be dynamic all the way.